cosmwasm/packages/wasmi-runtime/src/registration/cert.rs

```diff
@@ -347,6 +341,40 @@ pub fn verify_ra_cert(cert_der: &[u8]) -> SgxResult<Vec<u8>> {
     Ok(report_public_key)
 }
 
+#[cfg(all(feature = "SGX_MODE_HW", feature = "production"))]
```

Comment:
isn't this feature supposed to be added to `Cargo.toml`?

Reply:
Do you refer to the `production` feature? It's already in:
https://github.com/enigmampc/SecretNetwork/blob/system-tests-in-docker/cosmwasm/packages/wasmi-runtime/Cargo.toml#L16

Reply:
oh, i must've missed it